apps/protocol-frontend/src/components/ContributionMembersPie.tsx

```diff
@@ -0,0 +1,138 @@
+import { Flex, Heading, Text, useBreakpointValue } from '@chakra-ui/react';
+import { ResponsivePie } from '@nivo/pie';
+import { GovrnSpinner } from '@govrn/protocol-ui';
+
+type ContributionMembersCount = {
+  count: number;
+  display_name?: string | null | undefined;
+  user_id?: number | null | undefined;
+};
+interface ContributionTypesPieProps {
+  contributionMembersData: ContributionMembersCount[] | undefined;
+  isFetching: boolean;
+  isLoading: boolean;
+  isError: boolean;
+}
+
+const brandColorMap = [
+  '#ffb4e2',
+  '#fb84ce',
+  '#f854ba',
+  '#f526a6',
+  '#db0f8d',
+  '#ab076d',
+  '#76024e',
+];
+
+const ContributionMembersPie = ({
+  contributionMembersData,
+  isFetching,
+  isLoading,
+  isError,
+}: ContributionTypesPieProps) => {
+  const isMobile = useBreakpointValue({ base: true, lg: false });
+
+  const contributionsDataMap = contributionMembersData?.map(contribution => {
+    return {
+      id:
+        contribution.display_name !== null
```

Comment:
Could display_name be undefined as well?

Reply:
I'll add a protection for that too -- I think they're null if not set, but good to be safe